Wikipedia/Code/WMFHelpViewController.m

```diff
@@ -33,6 +41,16 @@ - (void)viewDidLoad {
     self.navigationItem.rightBarButtonItem = nil;
 }
 
+- (void)viewWillAppear:(BOOL)animated {
+    [super viewWillAppear:animated];
+    self.navigationController.toolbarHidden = NO;
+}
+
+- (void)webViewController:(WebViewController *)controller didTapOnLinkForArticleURL:(NSURL *)url {
+    WMFHelpViewController *articleViewController = [[WMFHelpViewController alloc] initWithArticleURL:url dataStore:self.dataStore];
```

Comment:
Link taps were using the regular `WMFArticleViewController` instead of `WMFHelpViewController`.